silkworm/db/kv/api/state_cache.hpp

```diff
@@ -134,10 +134,10 @@ class CoherentStateCache : public StateCache {
     void process_code_change(CoherentStateRoot* root, StateViewId view_id, const remote::AccountChange& change);
     void process_delete_change(CoherentStateRoot* root, StateViewId view_id, const remote::AccountChange& change);
     void process_storage_change(CoherentStateRoot* root, StateViewId view_id, const remote::AccountChange& change);
-    bool add(const KeyValue& kv, CoherentStateRoot* root, StateViewId view_id);
-    bool add_code(const KeyValue& kv, CoherentStateRoot* root, StateViewId view_id);
-    Task<std::optional<Bytes>> get(const Bytes& key, Transaction& txn);
-    Task<std::optional<Bytes>> get_code(const Bytes& key, Transaction& txn);
+    bool add(KeyValue&& kv, CoherentStateRoot* root, StateViewId view_id);
```

Comment:
I recommend to pass by value by default (and move at call sites), unless there's a need to disambiguate between overloads (e.g. `&&` vs `const&`).